lib/identity_cache.rb

```diff
@@ -89,8 +91,8 @@ def should_use_cache? # :nodoc:
     # == Parameters
     # +key+ A cache key string
     #
-    def fetch(key)
-      if should_use_cache?
+    def fetch(key, use_cache: should_use_cache?)
```

Comment:
Actually, my use case has to do with forcing the use of the cache for particular models in a circumstance in which the cache would otherwise be automatically bypassed. In particular, IdentityCache#fetch currently assumes that it should never use the cache if there is an open database transaction. I want to replace this assumption with logic that is specific to each model and, consequently, specific to each call to `fetch`.